<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>411</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actOpenDB"/>
   <addaction name="separator"/>
   <addaction name="actRecAppend"/>
   <addaction name="actRecInsert"/>
   <addaction name="actRecDelete"/>
   <addaction name="separator"/>
   <addaction name="actSubmit"/>
   <addaction name="actRevert"/>
   <addaction name="separator"/>
   <addaction name="actFields"/>
   <addaction name="actQuit"/>
  </widget>
  <action name="actOpenDB">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/images/open3.bmp</normaloff>:/icons/images/open3.bmp</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开数据库</string>
   </property>
  </action>
  <action name="actRecAppend">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/images/316.bmp</normaloff>:/icons/images/316.bmp</iconset>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
   <property name="toolTip">
    <string>添加记录</string>
   </property>
  </action>
  <action name="actRecInsert">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/images/306.bmp</normaloff>:/icons/images/306.bmp</iconset>
   </property>
   <property name="text">
    <string>插入</string>
   </property>
   <property name="toolTip">
    <string>插入记录</string>
   </property>
  </action>
  <action name="actRecDelete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/images/delete1.bmp</normaloff>:/icons/images/delete1.bmp</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
   <property name="toolTip">
    <string>删除记录</string>
   </property>
  </action>
  <action name="actSubmit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/images/save1.bmp</normaloff>:/icons/images/save1.bmp</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存修改</string>
   </property>
  </action>
  <action name="actRevert">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/images/ubdo.bmp</normaloff>:/icons/images/ubdo.bmp</iconset>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="toolTip">
    <string>取消修改</string>
   </property>
  </action>
  <action name="actFields">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/images/124.bmp</normaloff>:/icons/images/124.bmp</iconset>
   </property>
   <property name="text">
    <string>字段列表</string>
   </property>
   <property name="toolTip">
    <string>字段列表</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/images/exit.bmp</normaloff>:/icons/images/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
